Changed h1 to Healing

smellular opened the pull request:
Hello @stevenhauser

Commit:
Changed h1 to Healing

smellular commented:
And now I know how to get ahold of you quite easily-uru

Commit:
Ron's initial updates

- Changed “Yoga” to “Healing”
- Changed email
- Added Beginner Course Page
- Added text-shadow to h1, h2, h3
- Changed a:hover to text-shadow

smellular commented:
Hello @stevenhauser. I've made some changes and would like you to review them at your leisure. Hopefully they're not all :shit:.

stevenhauser commented:
Will try to check it out tomorrow @smellular

On Tuesday, February 23, 2016, smellular notifications@github.com wrote:

> Hello @stevenhauser https://github.com/stevenhauser. I've made some
> changes and would like you to review them at your leisure. Hopefully
> they're not all [image: :shit:].
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/reneesharpless/reneesharpless.github.io/pull/1#issuecomment-188028701
> .

stevenhauser commented on beginners-yoga-course.html:
```diff
@@ -0,0 +1,79 @@
+          <span class="unbreakable">A five week course beginning April 13th – May 11th.</span>
+          <span class="unbreakable"></span>
+        </p>
+        <p class="meta__item">Wedensdays 6:30-7:30pm</p>
```
JUST WHAT THE _HELL_ IS THIS TYPO???

stevenhauser replied:
Also en dash here.

stevenhauser commented on beginners-yoga-course.html:
```diff
@@ -0,0 +1,79 @@
+  <title>Prema Healing Cleveland</title>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Signika:300">
+  <link rel="stylesheet" href="main.css">
+  <style type="text/css"></style>
```
Smells like leftovers.

stevenhauser commented on beginners-yoga-course.html:
```diff
@@ -0,0 +1,79 @@
+
+      <div class="meta uppercase">
+        <p class="meta__item">
+          <span class="unbreakable">A five week course beginning April 13th – May 11th.</span>
```
Minor, but any `-` between dates and times should be an en dash. You make it by hitting `option + -` on your keyboard. It's slightly longer than a hyphen. And yes, I'm being serious because I am insane.

stevenhauser replied:
I'm not sure you want this to be `unbreakable` either since on small screens it may need to break.

stevenhauser commented on beginners-yoga-course.html:
```diff
@@ -0,0 +1,79 @@
+      <div class="meta uppercase">
+        <p class="meta__item">
+          <span class="unbreakable">A five week course beginning April 13th – May 11th.</span>
+          <span class="unbreakable"></span>
```
Smells like leftovers.

stevenhauser commented on beginners-yoga-course.html:
```diff
@@ -0,0 +1,79 @@
+      </div>
+
+      <div class="content">
+        <p>This course is a step-by-step introduction into Sivananda classical hatha yoga. Students will experience the Five Points of Yoga including proper exercise, breathing, diet, relaxation, and meditation. Sivananda yoga guides students through asanas, while relaxing between each posture. Classes begin with chanting, pranayama breathing, sun salutations and go into the 12 basic asanas. This ...
```
A few questions/notes:
1. Is "introduction into" the right phrase here or is "introduction to"?
2. Should "Sivananda classical hatha yoga" all be capitalized?

stevenhauser replied:
Also in general you want to avoid using `br`s for stylistic purposes. If the content itself actually logically breaks (for instance the lines of an address) then it might make sense. Not really a big deal though.

stevenhauser commented on beginners-yoga-course.html:
```diff
@@ -0,0 +1,79 @@
+
+      <div class="content">
+        <p>This course is a step-by-step introduction into Sivananda classical hatha yoga. Students will experience the Five Points of Yoga including proper exercise, breathing, diet, relaxation, and meditation. Sivananda yoga guides students through asanas, while relaxing between each posture. Classes begin with chanting, pranayama breathing, sun salutations and go into the 12 basic asanas. This ...
+        <p><b>REGISTER BY APRIL 1ST TO SAVE YOUR SPOT!</b></p>
```
WHY ARE YOU YELLING AT ME?

Challenge: can you capitalize this using only CSS? What are the pros/cons to doing so? 

Also I might suggest an `strong` instead of a `b`. Minor thing, I don't feel like getting into details.

stevenhauser commented on beginners-yoga-course.html:
```diff
@@ -0,0 +1,79 @@
+      <div class="content">
+        <p>This course is a step-by-step introduction into Sivananda classical hatha yoga. Students will experience the Five Points of Yoga including proper exercise, breathing, diet, relaxation, and meditation. Sivananda yoga guides students through asanas, while relaxing between each posture. Classes begin with chanting, pranayama breathing, sun salutations and go into the 12 basic asanas. This ...
+        <p><b>REGISTER BY APRIL 1ST TO SAVE YOUR SPOT!</b></p>
+        <p><b>Course is limited to 10 students per course.</b></p>
```
Same `strong` comment here.

stevenhauser commented on beginners-yoga-course.html:
```diff
@@ -0,0 +1,79 @@
+      <span class="headline__line pageHeader__secondary">Cleveland</span>
+    </h1>
+  </header>
+</div>
```
It's pretty common for the site "logo" to link back to the homepage. Now that the site has multiple pages, you might consider doing that.

stevenhauser replied:
I'd also note that on narrower screens, the header isn't centered on the page which looks strange against the centered content below. On the home page it wasn't as weird because all of the text was left aligned and it used to say "Prema Yoga Cleveland" which aligned nicely. So you might consider reworking the header to align better (by shrinking "Prema Healing" or enlarging "Cleveland" or something else).

Looks weird to me:
![image](https://cloud.githubusercontent.com/assets/963631/13287100/35fb9e4a-dad3-11e5-95a1-62d3a50e8d23.png)

Looks fine to me:
![image](https://cloud.githubusercontent.com/assets/963631/13287107/422e6f1c-dad3-11e5-9490-0e3d5a825d4d.png)

Looks very awkward to me:
![image](https://cloud.githubusercontent.com/assets/963631/13287115/4c18b8b6-dad3-11e5-9fc6-96eb378274e7.png)

stevenhauser commented on beginners-yoga-course.html:
```diff
@@ -0,0 +1,79 @@
+      <section class="additionalInfo">
+        <p class="headline inverted"><a href="https://docs.google.com/forms/d/1TUbJxIprhOcw02V6gS4UUgb7r8Kxm6rBtWOteg62ddw/viewform">Sign Up Here!</a></p>
+        <p class="headline inverted"><a href="index.html">[Back to Home Page]</a></p>
+      </section>
```
I would argue that neither `p` tag here is a `.headline` so you might consider handling those in a different way by creating a new class or something.

stevenhauser commented on index.html:
```diff
@@ -33,11 +34,8 @@ <h2 class="headline inverted uppercase program__title">Yoga</h2>
         <p class="meta__item">
-          <span class="unbreakable">Wednesdays 6:00 — 7:30pm</span>
-          <span class="unbreakable">@ Unity Spiritual Center</span>
+          <span class="unbreakable"><a href="beginners-yoga-course.html">Sign up here for the Beginners Yoga Course starting April 13th!</a></span>
```
Again, I wouldn't recommend this being `unbreakable` because on small screens the text needs to break. `unbreakable` is intended to be used for very short pieces of text to keep the words nicely grouped so that when text _does_ wrap, certain groupings of words stay together.

stevenhauser replied:
I'd also argue that as the primary call to action on the site, this doesn't stand out enough. To me it seems like a banner treatment might be more appropriate where it appears above the "Yoga" and "Reiki" headlines, spanning both columns. There are a lot of different ways to approach this, I just think that it needs more oomph.

stevenhauser commented on main.css:
```diff
@@ -28,17 +28,23 @@ body {
-  text-decoration: underline;
+  /*text-decoration: underline;*/
+  text-shadow: 2px 2px 5px #222;
+
```
I'd recommend removing the commented out line here. Git allows you to see changes over time, so there's rarely a need to leave commented code.

Also minor thing, but you have an extra blank line here that can be removed.

stevenhauser commented on main.css:
```diff
@@ -112,10 +118,12 @@ h1.headline {
 
 h2.headline {
   font-size: 2.1rem;
+  text-shadow: 2px 2px 5px #222;
```
This shadow is pretty repetitious. There are several ways of handling this and perhaps you've considered all of them:
1. Do what you did and repeat the shadow
2. Create a compound selector like `.headline, .pageHeader__primary { }`
3. Create a new class and apply it to the HTML
4. Others

I think what you've done is fine, I'm just trying to make you aware of other possibilities. Once you start learning Sass, you'll see other ways to reduce this duplication through things like variables and mixins.

stevenhauser commented:
I made a lot of comments here, but overall this looks pretty good and most of my comments are minor. 

One thing to remember with pull requests is that other people are always going to have comments on your work and how they might do something different, but you don't have to agree with them. So feel free to address the comments you want to and give the old finger to the others.

Commit:
Content, responsive styling, footer changes

- Made .program__row wider to accommodate new content when >60em
-  “unbreakable” class only applies when viewport is >60em (“Sign up
here for Beginners Yoga…” was not wrapping with the rest of the text in
mobile view )
- Beginners Course page no longer centered
- Cert. and Exp. sections updated
- Added tel. #
- Added links to footer
- Footer links display as block in mobile view
- Beg. course signup link opens in new tab

Commit:
per Steven's comments

- (How) Lo (can you) go?
- Page not responding
- Threw out the leftovers
- Will fix the typo next Wedensday
- One text-shadow to rule them all
- en(ded up fixing the) dashes
- Broke the “Unbreakable”s
- Country Grammar
- SORRY FOR YELLING (challenge complete, Pros/Cons you say?)
- Mas fuerte
- No comment
- No more Banner blues

stevenhauser commented:
I see @smellular has been busy. I can check out your changes after work if you'd like.

stevenhauser commented on beginners-yoga-course.html:
```diff
@@ -0,0 +1,82 @@
+      </div>
+
+      <section class="additionalInfo">
+        <p class="inverted"><a href="https://docs.google.com/forms/d/1TUbJxIprhOcw02V6gS4UUgb7r8Kxm6rBtWOteg62ddw/viewform" target="_blank">Sign Up Here!</a></p>
```
Small thing but the `inverted` class doesn't look necessary here since the `a` will already be Walter White.

stevenhauser commented on index.html:
```diff
@@ -86,11 +79,11 @@ <h2 class="headline inverted uppercase program__title">Reiki</h2>
-          <li>Reiki I, II, III — The Ideal Method — Westlake, OH</li>
-          <li>Shamanic Reiki Master — Mary Maynard — Westlake, OH</li>
+          <li>Shamanic Reiki Master – Mary Maynard – Westlake, OH</li>
+          <li>Animal Communication &amp; Reiki – The Ideal Method – Westlake, OH</li>
```
Mother of god

stevenhauser commented on index.html:
```diff
@@ -100,13 +93,13 @@ <h3 class="headline inverted">Certifications &amp; Experience</h3>
-      </a>
+      Contact
+      <a href="mailto:renee@premahealingcleveland.com">renee@premahealingcleveland.com</a> |
+      <a href="tel:2162824497">216.282.4497</a>
```
WELLLLLL look at that `tel:`. Very nice.

stevenhauser commented on main.css:
```diff
@@ -60,6 +60,13 @@ a:hover {
   text-transform: uppercase;
 }
 
+@media screen and (min-width: 60em) {
```
This change doesn't make a lot of sense to me. I think that keeping `unbreakable` unbreakable _all the time_ is a little easier to understand and is more predictable. I'm curious what specifically warranted this change and maybe we can come up with a better way to handle it.

stevenhauser commented on main.css:
```diff
@@ -123,6 +131,28 @@ h3.headline {
+
+.banner {
+  padding: 1em;
+  width: 360px;
```
I don't think this is desirable. Let's come up with a better way to handle this.

stevenhauser commented on main.css:
```diff
@@ -123,6 +131,28 @@ h3.headline {
+  .banner {
+    margin: 0 auto 30px;
+    padding: 0 1em;
+    width: 520px;
```
Same comment—let's come up with a better way to handle this.

stevenhauser commented on main.css:
```diff
@@ -123,6 +131,28 @@ h3.headline {
+    width: 520px;
+    border-right: 2px solid hsl(48, 89%, 60%);
+    border-left: 2px solid hsl(48, 89%, 60%);
+    border-radius: 5px;
```
This is a very interesting effect. I thought at first you used pseudo elements. :clap:

smellular replied:
I know. I am REALLY good at this!!!

stevenhauser commented on main.css:
```diff
@@ -142,15 +172,18 @@ h3.headline {
 .pageHeader__secondary {
-  letter-spacing: 0.68em;
-  margin-right: -0.68em;
+  letter-spacing: 1.05em;
```
The name here is a mistake on my part. `pageHeader__secondary` should be `pageHeader--secondary`. I might fix this and any similar mistakes in a future PR.

stevenhauser commented on main.css:
```diff
@@ -163,6 +196,7 @@ h3.headline {
 @media screen and (min-width: 60em) {
   .program__row {
     margin-bottom: 3rem;
+    width: 975px;
```
This seems funky to me. Let's discuss what you're intending to accomplish with this.

stevenhauser commented on main.css:
```diff
@@ -194,6 +229,10 @@ h3.headline {
 
+.vertical-padding {
+  padding: 20px 0 0;
+}
```
Not sure about this.

stevenhauser commented on main.css:
```diff
@@ -227,3 +266,17 @@ h3.headline {
   margin-right: 1em;
 }
+
+@media screen and (min-width:1em) {
```
This media query seems a little funky.

stevenhauser commented on main.css:
```diff
@@ -227,3 +266,17 @@ h3.headline {
+
+}
+
+@media screen and (max-width: 420px) {
```
Same here, seems semi-ass-backwards.

Commit:
Minor <head> changes

some clean up

Commit:
Adjusted per Esteban's commentos

All da fings

Commit:
Make the page title work better on small screens

Commit:
Rework the call to action on beginners yoga page

- Make the action more dominant by giving it a button-like style
- Move it around

Note that there's a minor specificity issue here which is why
the selector needs to be qualified with an `a` tag. Since this
is a small site, I'm not too :corn:cerned by it.
